src/signs/ui/search/Signs.css

```diff
@@ -19,7 +19,10 @@
   border-style: solid;
   border-width: 0 0 0.5px 0;
 }
-
+.HangingIndent {
+  padding-left: 44px;
+  text-indent: -44px;
+}
```

Comment:
The coding conventions tell to use BEM naming. But they have been violated many times. Do you think we should drop the convention(s)?

Reply:
no i am going to read up again on them.